### DISCUSSION TITLE
Changing the nickname leads to errors in the app

### DISCUSSION THREAD

**Discussion by kseirru** [community/community#61410] [Mobile]
### Select Topic Area

Bug

### Body

If you change the username of the profile on github, while not logging into the account in the mobile application, then you cannot use the "Starred" item in the "My Work" menu on the main menu, writes: "Could not resolve to User with the login of 'Here old username'"

**Comment by the-officialjosh** [comment]
It might be an issue with the caching maybe try to login on the browser and see if it is resolved

**Comment by h7x4** [comment]
This is such an interesting bug. I changed my username probably over a year ago, and I hit this bug for the first time when I tried to add a project to a star list the other day. I bet logging out and logging in again would fix the issue, but I kept it like this for a while just to play around with it to see where there are limitations. 

Up until now, I've only found the following:

- Adding a project to a list is failing
- Settings -> Accounts - lists my old username
- Browsing starred repositories is broken

**Reply by h7x4** [reply]
It also seems that there are a bunch of duplicate discussions, relating to this
